docs/configuration.md

```diff
@@ -35,12 +35,14 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `INPUT_SIZE_LIMIT`         | "1mb"            | The [koa-jsonrpc](https://github.com/Bitclimb/koa-jsonrpc) maximum size allowed for requests                                                                                                                            |
 | `MAX_BLOCK_RANGE`          | "5"              | The maximum block number greater than the mirror node's latest block to query for                                                                                                                                       |
 | `OPERATOR_ID_MAIN`         | ""               | Operator account ID used to pay for transactions. In `S.R.N` format, e.g. `0.0.1001`.                                                                                                                                   |
+| `OPERATOR_KEY_FORMAT`        | "DER"               | Operator private key format. Valid types are `DER`, `HEX_ECDSA`, or `HEX_ED25519` |
```

Comment:
This might not be entirely true when it says that the valid types are `DER`, `HEX_ECDSA`, or `HEX_ED22519`. Because [here](https://github.com/hashgraph/hedera-json-rpc-relay/pull/2591/files#diff-e959cfa5abcb8b82e4d8d85f4476d00f788c50a80e6e9081b390f09082f58ad2R257) is a `default` block and will read the operator key as DER type. This means `OPERATOR_KEY_FORMAT` is not necessary be `DER`, `HEX_ECDSA`, or `HEX_ED25519` because the `initPrivateKey` function will accept any arbitrary values. `OPERATOR_KEY_FORMAT` can be "HEDERA", or "HBAR", or "swirlds-labs", "bird", etc. and the `initPrivateKey` will still read the `OPERATOR_KEY_MAIN` as DER.

Reply:
Okay, yes, will update.

Reply:
Handled.